prog/vm/nexus.rb

```diff
@@ -207,20 +207,20 @@ def before_run
           [[], [], [], [frame["force_host_id"]], []]
         elsif vm.location_id == Location::GITHUB_RUNNERS_ID
           runner_location_filter = [Location::GITHUB_RUNNERS_ID, Location::HETZNER_FSN1_ID, Location::HETZNER_HEL1_ID]
-          runner_location_filter.append(Location::LEASEWEB_WDC02_ID) if vm.vcpus == 60
           runner_location_preference = [Location::GITHUB_RUNNERS_ID]
           runner_family_filter = [vm.family]
-          if runner
-            if (preferred_locations = runner.installation.project.get_ff_preferred_runner_locations) && !preferred_locations.empty?
-              # Just be sure your preferred ones are also in the location filter
-              runner_location_filter |= preferred_locations
-              runner_location_preference = preferred_locations
-            end
-            if vm.family == "standard" && runner.installation.project.get_ff_performance_runners
-              runner_family_filter = [vm.family, "performance"]
-            end
+          prefs = if runner
```

Comment:
could also be written as:
```
prefs = runner ? runner.installation.allocator_preferences : {}
```

Reply:
I remembered while rebasing why I chose the `if/else` block. I added more code to the if block in the PR. https://github.com/ubicloud/ubicloud/pull/3268/commits/5a9593bc0016c78daf19daba32f241e0972183d4